client/src/partials/reports_proposed/debtor_group/annual.js

```diff
@@ -0,0 +1,107 @@
+angular.module('bhima.controllers')
+.controller('DebtorGroupAnnualReportController', DebtorGroupAnnualReportController);
+
+DebtorGroupAnnualReportController.$inject = [ '$http', '$sce' ];
+
+/**
+* Debtor Group Annual PDF Report
+*
+* This report provides an overview of all debits and credits accrued by each
+* debtor group over the selected fiscal year.  This should only be done for
+* fiscal years that are closed, and therefore have closing balances posted.
+*
+* DEV NOTE
+*   This controller is largely based of the Balance Sheet PDF report, with a
+* slight code refresh (controller as syntax, etc).  If there are critical bugs
+* or updates to be made there, it is likely the case that they need to be
+* changed here as well.
+*/
+function DebtorGroupAnnualReportController($http, $sce) {
+  var vm = this;
+
+  vm.state = 'fresh';
+
+  // TODO -- is this defined in a central location somewhere?  Maybe a service?
+  // Configuration objects optionally passed to /report/build - drives configuration UI
+  vm.configuration = {
```

Comment:
There is service called ```reportConfigService``` it provides data like configuration and this service must be used by all pdf report until now.